stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -141,7 +142,7 @@ public static InspectorModulesProvider defaultInspectorModulesProvider(final Con
         modules.add(new DOMStorage(context));
         modules.add(new HeapProfiler());
         modules.add(new Inspector());
-        modules.add(new Network(context));
+        modules.add(new Network(context, DefaultAsyncPrettyPrinterRegistry.getInstance()));
```

Comment:
To confirm I understand correctly with regard to the last suggestion, Network will have methods register and unregister which will internally ask async pretty printer registry to do the registering and unregistering?